docs/UnityGroceriesRealWorld.md

```diff
@@ -0,0 +1,44 @@
+# UnityGroceries-RealWorld Dataset
+
+As part of this project, we collected a real world dataset containing 1267 images of the 63 corresponding grocery items. This dataset is primary used to compare performance of model trained on the Synthetic counterparts. This dataset can be downloaded [here](https://storage.googleapis.com/datasetinsights/data/groceries/v3.zip).
+
+## dataset collection
+
+To take the photos for the UnityGroceries-RealWorld dataset, we devised a strategy to vary object selection, placement, lighting, and background. After a few test shoots, we realized that major barriers to randomness of placement and object selection were the actual humans doing the selecting and placing. Hence, planning on how to randomize while taking photos became highly important, and something we did before taking photos.
```

Comment:
>After a few test shoots, we realized that major barriers to randomness of placement and object selection were the actual humans doing the selecting and placing. Hence, planning on how to randomize while taking photos became highly important, and something we did before taking photos.

I don't understand this part. What was the issue you discovered? If you want to include this section, be more specific about the previous and new method.

Reply:
@SCRktkt I am also a little confused about this part. It seems this sentence can be safely removed, as the "placement strategy" have be described in detail below.

Reply:
We found that we, the humans, needed more humans and a better strategy to randomize more thoroughly and faster. This new strategy is what is outlined.